packages/nimble-components/src/utilities/tests/storybook.ts

```diff
@@ -156,6 +156,7 @@ export const incubatingWarning = (config: IncubatingWarningConfig): string => `
 #incubating-warning {
     color: red;
     font: var(${bodyFont.cssCustomProperty});
+    padding-bottom: 16px;
```

Comment:
Adding some padding to the incubating warning, so it's not so squished. This is responsible for the Chromatic diffs.